CesiumGeospatial/include/CesiumGeospatial/LocalHorizontalCoordinateSystem.h

```diff
@@ -72,6 +72,30 @@ class CESIUMGEOSPATIAL_API LocalHorizontalCoordinateSystem {
       double scaleToMeters = 1.0,
       const Ellipsoid& ellipsoid = Ellipsoid::WGS84);
 
+  /**
+   * @brief Create a new coordinate system with a specified transformation to
+   * the Earth-Centered, Earth-Fixed frame. This is an advanced constructor and
+   * should be avoided in most cases.
+   *
+   * @param localToEcef The transformation matrix from this coordinate system to
+   * the ECEF frame.
+   */
+  explicit LocalHorizontalCoordinateSystem(const glm::dmat4& localToEcef);
+
+  /**
+   * @brief Create a new coordinate system with the specified transformations
+   * between the local frame and the Earth-Centered, Earth-Fixed frame. This is
+   * an advanced constructor and should be avoided in most cases.
+   *
+   * @param localToEcef The transformation matrix from this coordinate system to
+   * the ECEF frame. This must be the inverse of `ecefToLocal`.
+   * @param ecefToLocal The transformation matrix from the ECEF frame to this
+   * coordinate system. This must be the inverse of `localToEcef`.
+   */
+  LocalHorizontalCoordinateSystem(
+      const glm::dmat4& localToEcef,
+      const glm::dmat4& ecefToLocal);
+
```

Comment:
I appreciate the direct access to the members from the ctor.
- Is it possible to mention a use case in the comments that this would be typically used for?
- Also, this states that the two matrices (`ecefToLocal localToEcef`) must be inverse of each other, but there doesn't seem to be any code in the class that enforces this. Maybe _must_ really means _typically is_

Reply:
> Is it possible to mention a use case in the comments that this would be typically used for?

Ok, I added a note.

> Also, this states that the two matrices (ecefToLocal localToEcef) must be inverse of each other, but there doesn't seem to be any code in the class that enforces this. Maybe must really means typically is?

No, they must be. It's a precondition. Enforcing it would be costly and unnecessary. It doesn't make any sense to have two transformations that are meant to be the opposite of each other yet one is not the inverse of the other. I tweaked the doc to make this clearer. I considered adding an assertion, but that quickly gets into floating point equality land and it's just not worth the trouble.